### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by deepakjangid-MSFT**
Update docs for stateful service example and folder hierarchy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,19 @@ This repository contains a set of simple sample projects to help you getting sta
...
 ## How the samples are organized
 
-The samples are divided by the category and [Service Fabric programming model][service-fabric-programming-models] that they focus on: Reliable Actors, Reliable Services. Most real applications will include a mixture of the concepts and programming models. Note that the support for reliable stateful service is not there today and will come in near future.
-
-## Actor samples
+The samples are divided by the category and [Service Fabric programming model][service-fabric-programming-models] that they focus on: Reliable Actors, Reliable Services. Most real applications will include a mixture of the concepts and programming models.
+
+### Folder Hierarchy
+src/ - Source of the application divided by different modules by sub-folders.  
+&lt;Application Folder&gt;/ - Service Fabric Application folder heirarchy. After compilation the executables are places in code   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;subfolders.
```
Comment (raunakpandya) [inline README.md]: Should we change folder to package?

Comment (raunakpandya) [inline README.md]: Instead of adding these spaces, should we use bullet/ hyphen for the items on left?

Comment (deepakjangid-MSFT) [reply README.md]: Application Package Folder makes more sense. What do you think?

Comment (deepakjangid-MSFT) [reply README.md]: The spaces are to align the multi-line explanation. Adding bullet won't help in this case.
I can add them for all the folders though to make it look more clean. Let me try.

Comment (raunakpandya) [reply README.md]: ok.. keep a, p and f small case only..

File: README.md [hunk 2]
```diff
@@ -10,9 +10,19 @@ This repository contains a set of simple sample projects to help you getting sta
...
 ## How the samples are organized
 
-The samples are divided by the category and [Service Fabric programming model][service-fabric-programming-models] that they focus on: Reliable Actors, Reliable Services. Most real applications will include a mixture of the concepts and programming models. Note that the support for reliable stateful service is not there today and will come in near future.
-
-## Actor samples
+The samples are divided by the category and [Service Fabric programming model][service-fabric-programming-models] that they focus on: Reliable Actors, Reliable Services. Most real applications will include a mixture of the concepts and programming models.
+
+### Folder Hierarchy
+* src/ - Source of the application divided by different modules by sub-folders.  
+* &lt;application package folder&gt;/ - Service Fabric Application folder heirarchy. After compilation the executables are placed in code subfolders.
```
Comment (raunakpandya) [inline README.md:17]: Some miscommunication here.. How does the below look like? 
* &lt;Application Name&gt;/ - This constitutes the Service Fabric application package which can be deployed on cluster. After compilation the binaries are placed in the code subfolder.

File: README.md [hunk 3]
```diff
@@ -10,9 +10,19 @@ This repository contains a set of simple sample projects to help you getting sta
...
-
-## Actor samples
+The samples are divided by the category and [Service Fabric programming model][service-fabric-programming-models] that they focus on: Reliable Actors, Reliable Services. Most real applications will include a mixture of the concepts and programming models.
+
+### Folder Hierarchy
+* src/ - Source of the application divided by different modules by sub-folders.  
+* &lt;application package folder&gt;/ - Service Fabric Application folder heirarchy. After compilation the executables are placed in code subfolders.  
+* build.sh - Script to build source on Linux shell.  
+* build.ps1 - PowerShell script to build source on Windows.  
+* install.sh - Script to install Application from Linux shell.
```
Comment (raunakpandya) [inline README.md:20]: lower case application

File: README.md [hunk 4]
```diff
@@ -10,9 +10,19 @@ This repository contains a set of simple sample projects to help you getting sta
...
+
+### Folder Hierarchy
+* src/ - Source of the application divided by different modules by sub-folders.  
+* &lt;application package folder&gt;/ - Service Fabric Application folder heirarchy. After compilation the executables are placed in code subfolders.  
+* build.sh - Script to build source on Linux shell.  
+* build.ps1 - PowerShell script to build source on Windows.  
+* install.sh - Script to install Application from Linux shell.  
+* install.ps1 - PowerShell script to install application from Windows.  
+* uninstall.sh - Script to uninstall application from Linux shell.  
+* uninstall.ps1 - PowerShell script to unintall application from Windows.
```
Comment (raunakpandya) [inline README.md:23]: Should we add dotnet-include.sh as well here for explanation?

**Review by raunakpandya — APPROVED** [review]

**Pull request merged**